tests/code/test_addRemoveInterest.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from app.logic.addRemoveInterest import addRemoveInterest
+from flask import json
+
+@pytest.mark.integration
+def test_addRemoveInterest():

```

Comment:
This misses the case where addRemoveInterest doesn't do what you want. How could the function fail?

Reply:
We've added some more test cases